bin/token/create.js

```diff
@@ -119,6 +119,11 @@ module.exports = Cli.createCommand('create', {
 
 function handleTokenCreate(args) {
     var profile = args.profile;
+
+    if (profile.securityVersion !== 'v1') {
+        throw Cli.error.invalid('The `wt token create` command is not supported in the target service security configuration.');
```

Comment:
The `wt token create` command is not supported with the given security configuration of the target service.